feat: custom values to sandbox iframe (#29590)
apache/superset: superset-embedded-sdk/README.md

  ]
}
```

> ### Sandbox Iframe

Reviewer: ```suggestion
### Sandbox iframe
```

Author: change done

> Embedded SDK by default create an iframe with [sandbox](https://developer.mozilla.org/es/docs/Web/HTML/Element/iframe#sandbox) mode
> which applies extra restrictions to the content in the frame, for some cases default sandbox values are not enough
> you can pass `iframeSandboxExtras` to add additional values for example

Reviewer: ```suggestion
The Embedded SDK creates an iframe with [sandbox](https://developer.mozilla.org/es/docs/Web/HTML/Element/iframe#sandbox) mode by default 
which applies certain restrictions to the iframe's content. To pass additional sandbox attributes you can use `iframeSandboxExtras`:
```

Author: change done
